spec/lib/karafka_spec.rb

```diff
@@ -3,82 +3,15 @@
 RSpec.describe Karafka do
   subject { described_class }
 
-  let(:std_logger) { subject.logger }
-
-  before do
-    std_logger
-  end
-
-  after do
-    subject.logger = std_logger
-  end
-
-  describe '#logger=' do
-    let(:logger) { double }
-
-    it 'assigns logger' do
-      subject.logger = logger
-      expect(subject.instance_variable_get(:'@logger')).to eq logger
+  describe '.logger' do
+    it 'expect to use app logger' do
+      expect(subject.logger).to eq described_class::App.config.logger
```

Comment:
@mensfeld I am not pretty sure, but maybe if we use `described_class` as a `subject` so we should use `subject::App.config.logger` and `subject::App.config.monitor` in the test below. What do  you think?